doc/api/net.md

```diff
@@ -40,8 +40,10 @@ added: v0.1.90
 
 * {Error}
 
-Emitted when an error occurs.  The [`'close'`][] event will be called directly
-following this event.  See example in discussion of `server.listen`.
+Emitted when an error occurs. Unlike [`net.Socket`][], the [`'close'`][]
+event will **not** be emitted directly following this event unless
+`server.close()` is manually called. See the example in discussion of
+`server.listen()`.
```

Comment:
@sam-github I'd tried to link server.close() and server.listen() but failed. Now i know the reason, i have modified doc again.